esrally/track/track.py

```diff
@@ -450,63 +450,64 @@ def admin_op(self):
     @classmethod
     def from_hyphenated_string(cls, v):
```

Comment:
I think `R0911` is unjustified for a method like this. I vote to ignore it here and keep the old style.